Shared/Supporting Files/Views/CategoryView.swift

```diff
@@ -0,0 +1,52 @@
+// Copyright 2022 Esri
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//   https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+import SwiftUI
+
+struct CategoryView: View {
+    /// A Boolean value that indicates whether the user is searching.
+    @Environment(\.isSearching) private var isSearching
+    
+    /// All samples retrieved from the Samples directory.
+    let samples: [Sample]
+    
+    /// The search query in the search bar.
+    @Binding var query: String
+    
+    /// A Boolean value that indicates whether to present the about view.
+    @State private var isAboutViewPresented = false
+    
+    var body: some View {
+        VStack {
```

Comment:
If the VStack is only for grouping the conditional views and no other purpose, use [`Group`](https://developer.apple.com/documentation/swiftui/group) instead